### DISCUSSION TITLE
Include title in api.json

### DISCUSSION THREAD

**Discussion by zamp** [comfyanonymous/ComfyUI#1345] [Ideas]
I'm creating a tool that runs ComfyUI by sending workflow_api.json to it. I overwrite values in the json before sending it but now that the only way to know which node is which based on a number it's really tedious to manually input these numbers in every time I change the workflow.
It would be a piece of cake if the title was included in the api.json and I could just search for a specific title instead of rely on the id.

**Comment by wcurti** [comment]
Yep, I agree. This would be another solution to the [same problem.](https://github.com/comfyanonymous/ComfyUI/issues/1326#issue-1866991031)
